index.html

```diff
@@ -0,0 +1,389 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SHOPNO!</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700&family=Montserrat:wght@400;700&family=Raleway&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="shortcut icon" href="#">
+</head>
+
+<body>
+    <header class="header">
+        <div class="container">
+            <div class="header_logo">
+                <img src="./images/logo.png" alt="logo" class="logo">
+            </div>
+            <h1 class="header_text__style">

```

Comment:
please, follow BEM naming convention. it has to be `header__text-style`. but you can delete `style` part at all, it doesn't make sense

Reply:
Thanks!
Deleted everywhere, done by BM